src/GraphCanvas.tsx

```diff
@@ -120,7 +120,8 @@ export const GraphCanvas: FC<GraphCanvasProps & { ref?: Ref<GraphCanvasRef> }> =
       const canvasRef = useRef<HTMLCanvasElement | null>(null);
 
       useImperativeHandle(ref, () => ({
-        centerGraph: (n?: string[]) => rendererRef.current?.centerGraph(n),
+        centerGraph: (n?: string[], centerIfNodesNotInView?: boolean) =>
```

Comment:
I chose default to false because what this is actually doing is ONLY centering the nodes if their not in view. So if you called `centerGraph(nodes)` with this defaulted to true then it wouldn't always center

maybe I should rename the flag to somethin like `centerOnlyIfNodesNotInView`